packages/components/src/internal/components/domainproperties/list/ListDesignerPanels.tsx

```diff
@@ -268,6 +304,13 @@ class ListDesignerPanelsImpl extends React.PureComponent<Props & InjectedBaseDom
                     title="Importing data from selected file..."
                     toggle={submitting && file !== undefined}
                 />
+                <ConfirmImportTypes
+                    designerType="List"
+                    show={importErrorModalOpen}
+                    error={importError}
```

Comment:
same comment here, do you think we could just use something like `show={importError !== undefined}`?

Reply:
Fixed!